README.md

```diff
@@ -13,41 +13,29 @@ Windows      | Python 3.8      | [![TeamCity](https://dev.aegresco.com/teamcity/
-- Full whitelisting and blacklisting support for adding and excluding files and directories to projects
 
-The core architecture is much more stable and maintainable than old CSBuild's, and tools are easier to implement than
-they were before. The new architecture also successfully decouples csbuild from the c++ language, allowing it to be
-truly versatile and language-agnostic. The new csbuild also cuts down considerably on wasted time during initial startup.
-Now that the majority of the core features have been implemented, we expect feature parity for the tools and target platforms
-supported by old CSBuild to start coming online very quickly, shortly followed by solution generation, chunking,
-and the gui tools.
+The core architecture is much more stable and maintainable than old CSBuild's, and tools are easier to implement than they were before. The new architecture also successfully decouples csbuild from the c++ language, allowing it to be truly versatile and language-agnostic. The new csbuild also cuts down considerably on wasted time during initial startup. Now that the majority of the core features have been implemented, we expect feature parity for the tools and target platforms supported by old CSBuild to start coming online very quickly, shortly followed by solution generation, chunking, and the gui tools.
+
+Documentation hasn't been created for the new version of csbuild yet; however, we have created a large suite of tests, so in the short term, a lot of information can be gleaned from looking at the make.py file in each test.
 
 Code for old csbuild, for those interested in it, can be found here: https://github.com/SleepingCatGames/csbuild
 
 ---
 
 ## Development Notes
 
-Frozen requirements files are provided for both Python 2 & 3.  Currently, the only dependency is pylint which is only
-needed when running the pylint unit test.  If using virtual Python environments for csbuild development, please keep in
-mind that the pylint test will fail on macOS under the Python 2 virtual environment.  However, if you install pylint to
-your system-installed copy of Python 2.7 and run against that, the pylint unit test will pass.  This issue does not seem
-to be present on Windows or Linux and all other unit tests are unaffected by the presence of a Python2 virtual
-environment on macOS.
+Frozen requirements files are provided for both Python 2 & 3.  Currently, the only dependency is pylint which is only needed when running the pylint unit test.  If using virtual Python environments for csbuild development, please keep in mind that the pylint test will fail on macOS under the Python 2 virtual environment.  However, if you install pylint to your system-installed copy of Python 2.7 and run against that, the pylint unit test will pass.  This issue does not seem to be present on Windows or Linux and all other unit tests are unaffected by the presence of a Python2 virtual environment on macOS.
```

Comment:
Might be worth updating this block at some point to indicate that pylint has been disable for Python2 and our current stance on using csbuild with Python2.